cdap-app-fabric/src/test/java/co/cask/cdap/metadata/MetadataHttpHandlerTest.java

```diff
@@ -150,6 +174,16 @@ public void testProperties() throws Exception {
     );
     Assert.assertEquals(expected, searchProperties);
 
+    searchProperties = searchMetadata(Id.Namespace.DEFAULT, "multiword:w*", MetadataSearchTargetType.ALL);
+    Assert.assertEquals(2, searchProperties.size());
```

Comment:
It would be a good idea to assert the exact ids too, not just size. Same below too.